From 3af1c044ade19d346eec98899a275ec3547c11b3 Mon Sep 17 00:00:00 2001
From: Mikko Harju <mikko.harju@jolla.com>
Date: Tue, 6 Aug 2013 11:09:05 +0000
Subject: [PATCH 3/5] Fixed raw coordinate reporting for touch events

The raw coordinates were not reported by the evdevtouch plugin.

Change-Id: Ia06129edd771b12cc9f6a97dc678b545d226af70
Reviewed-by: Andy Nichols <andy.nichols@digia.com>

Upstream-Status: Pending
---
 src/platformsupport/input/evdevtouch/qevdevtouch.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/platformsupport/input/evdevtouch/qevdevtouch.cpp b/src/platformsupport/input/evdevtouch/qevdevtouch.cpp
index 2c93da1..ddd059c 100644
--- a/src/platformsupport/input/evdevtouch/qevdevtouch.cpp
+++ b/src/platformsupport/input/evdevtouch/qevdevtouch.cpp
@@ -421,6 +421,8 @@ void QEvdevTouchScreenData::processInputEvent(input_event *data)
             tp.normalPosition = QPointF((contact.x - hw_range_x_min) / qreal(hw_range_x_max - hw_range_x_min),
                                         (contact.y - hw_range_y_min) / qreal(hw_range_y_max - hw_range_y_min));
 
+            tp.rawPositions.append(QPointF(contact.x, contact.y));
+
             m_touchPoints.append(tp);
 
             if (contact.state == Qt::TouchPointReleased)
-- 
1.8.1.2

